config/cola.conf

```diff
@@ -0,0 +1,23 @@
+include "defaults.conf"
+JIANT_DATA_DIR="/Users/anhadmohananey/Desktop/NLPResearch/jiant/jiant/data"
```

Comment:
This shouldn't live in a conf file, even on your local machine. Use path_config.sh to set these.